one_command_install.sh

```diff
@@ -19,17 +19,18 @@ cd ${metAnnotateDir}/data
 echo "Checking Refseq database being downloaded"
 #This might take a few hours
 if [ ! -e "Refseq.fa" ] || [ ! -e "Refseq.fa.ssi" ]; then
-    echo "downloading refseq db, will take a few hours"
-    wget ftp://ftp.ncbi.nlm.nih.gov/refseq/release/complete/complete.nonredundant_protein*.protein.faa.gz
-    zcat *.faa.gz >Refseq.fa
-    echo "preprocessing Refseq.fa, removing uncommon amino acids"
+    echo "Downloading RefSeq database; could take a few hours if connection is slow"
+    # Note: when downloading from NCBI instead of Zenodo, this may have downloaded several files and then combined them. Keep this in mind if moving back to downloading directly from NCBI.
```

Comment:
Capital "W" on the when lol! 

> when downloading from NCBI instead of Zenodo, this may have downloaded several files and then combined them. Keep this in mind if moving back to downloading directly from NCBI.

When downloading what? The Refseq Database? 
Can you explain this better?

Reply:
Updated -- thoughts?